impeller/tools/build_metal_library.py

```diff
@@ -111,10 +111,17 @@ def main():
 
   command += args.source
 
-  subprocess.check_output(command, stderr=subprocess.STDOUT)
+  try:
+    subprocess.check_output(command, stderr=subprocess.STDOUT, text=True)
```

Comment:
The presubmits will tell, but hopefully CI has a recent enough python for `text=True`.

Reply:
I'm _pretty_ sure we do. I was going to use subprocess.run here but I'm not sure we have a recent enough for that. If it fails I'll change this.

Reply:
```
impeller/tools/build_metal_library.py:116:2: C0103: Variable name "e" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern (invalid-name)
```

For real?